install_config/registry/accessing_registry.adoc

```diff
@@ -60,14 +60,14 @@ container:
 the Docker registry:
 +
 ----
-# docker ps | grep ose-docker-registry
+# docker ps | grep registry
```

Comment:
This will give you more than desired (at least an instance of `origin-pod` or `ose-pod`). To limit the output just to the `docker-registry` container, you could write:

```
docker ps --filter=name=registry_docker-registry
# or even (to be sure to select containers from default namespace)
docker ps --filter=name=registry_docker-registry.*_default_
```